### PR TITLE
Remove decimals from the min/max input values in the fill component

### LINKED ISSUE #8011: [STYLES] Values generated when changing the size by value have too many decimals
like for example when styling the portland clients dataset by customer_value

### PR DESCRIPTION
**Pull request by javierarce**
Fixes #8011 

This PR remove decimals from the min/max input values in the fill component that were getting added because of [the method we use](https://github.com/CartoDB/cartodb/compare/8011-styles-decimals?expand=1#diff-535e96ef55f51f38739ebcb2373ecdc0L29) to calc those numbers from the fixed value in the form.

### CODE REVIEW

**Comment by javierarce** [comment]
CR: @xavijam

File: lib/assets/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-content-view.js [hunk 1]
```diff
@@ -26,11 +26,13 @@ module.exports = CoreView.extend({
...
   _calculateRangeFromFixed: function (fixed, percent) {
     percent = percent || 30;
+
     var span = this.options.max - this.options.min;
     var delta = fixed / span;
+
     return [
-      Math.max(this.options.min, fixed - percent * delta),
-      Math.min(this.options.max, fixed + percent * delta)
+      Math.floor(Math.max(this.options.min, fixed - percent * delta)),
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-content-view.js]: Why floor and not round?

Comment (javierarce) [inline lib/assets/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-content-view.js]: To stay safe and avoid increasing the number (eg: 5.85 -> 6)

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-content-view.js]: :+1:

**Comment by xavijam** [comment]
Dale Arce!!! :es:

**Pull request merged**